src/main/java/org/openelisglobal/menu/applicationpropertiescontroller/ApplicationPropertiesController.java

```diff
@@ -0,0 +1,61 @@
+package org.openelisglobal.menu.applicationpropertiescontroller;
+
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.http.MediaType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import java.util.Arrays;
+
+import java.util.Map;
+import java.util.HashMap;  
+
+@RestController
+@RequestMapping("/rest/properties")
+public class ApplicationPropertiesController {
+
+    @Autowired
+    private ConfigurableEnvironment env;
+
+    @GetMapping
+    public ResponseEntity<Map<String,String>> getProperties() {
+    Map<String,String> properties = new HashMap<>();
+    env.getPropertySources().forEach(propertySource -> {
+        if (propertySource instanceof MapPropertySource) {
+            MapPropertySource mapPropertySource = (MapPropertySource) propertySource;
+            Arrays.stream(mapPropertySource.getPropertyNames()).forEach(propertyName -> {
```

Comment:
@konavivekramakrishna  , since you are reading all the available properties ,
we need to exclude some specific sensitive properties from being exposed to the UI 
ie 
```
server.ssl.key-store = file:/etc/openelis-global/keystore
server.ssl.key-store-password = kspass
server.ssl.key-password = kspass
server.ssl.trust-store= file:/etc/openelis-global/truststore
server.ssl.trust-store-password=tspass
encryption.general.password=kspass
```